src/pages/index.tsx

```diff
@@ -115,12 +115,12 @@ export default function Home({
               <Box>
                 <Text lineHeight="180%" fontSize="md" mb={4}>
                   <Link
-                    href="/blog/2024/05/21/solidity-0.8.26-release-announcement/"
+                    href="https://soliditylang.org/blog/2024/09/04/solidity-0.8.27-release-announcement"
```

Comment:
```suggestion
                    href="/blog/2024/09/04/solidity-0.8.27-release-announcement/"
```
The link should be relative.

Reply:
Noticed that Nikola already replied, so resolving this.